lib/rules/prefer-node-protocol.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author Yusuke Iinuma
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const isBuiltinModule = require("is-builtin-module")
+const messageId = "preferNodeProtocol"
+
+module.exports = {
+    meta: {
+        docs: {
+            description:
+                "enforce using the `node:` protocol when importing Node.js builtin modules.",
+            recommended: true,
+            url: "https://github.com/eslint-community/eslint-plugin-n/blob/HEAD/docs/rules/prefer-node-protocol.md",
+        },
+        fixable: "code",
+        messages: {
+            [messageId]: "Prefer `node:{{moduleName}}` over `{{moduleName}}`.",
+        },
+        schema: [],
+        type: "suggestion",
+    },
+    create(context) {
```

Comment:
Would a combination of the [visit-require](https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/util/visit-require.js) and [visit-import](https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/util/visit-import.js) be able to work here? :thinking:

Reply:
I used `visit-require` and `visit-import` by https://github.com/eslint-community/eslint-plugin-n/pull/183/commits/a73ef36035218b939b177efcd8cc4fb90395d8fb